netbox/project-static/src/util.ts

```diff
@@ -409,7 +409,7 @@ export function createElement<
  * @returns Degrees in Fahrenheit.
  */
 export function cToF(celsius: number): number {
-  return celsius * (9 / 5) + 32;
+  return Math.round((celsius * (9 / 5) + 32 + Number.EPSILON) * 10) / 10;
```

Comment:
Does this actually work? Certain numbers cannot be represented exactly using floating point (0.1 is one example); rounding doesn't change this.

I would expect the problem is at the point of rendering it back to decimal.

Reply:
Can you please provide an example for this?

I checked it with `0.1` and `0.3` and still got the expected value
```javascript
Math.round((0.1 + Number.EPSILON) * 10) / 10; # 0.1
Math.round((0.3 + Number.EPSILON) * 10) / 10; # 0.3
```

Reply:
OK, I think I can replicate via Javascript console in Chrome:

![image](https://user-images.githubusercontent.com/44789/134034022-cd0ac30d-31b0-4f26-bb77-8832d01780f0.png)

The problem is indeed fixed by your method, but actually it would be simpler to multiply by 9 and then divide by 5, rather than multiply by (9/5) which is an inexact floating point value; it appears that the error in (9/5) is magnified by multiplying by the degrees C value.

Reply:
There's no guarantee that doing *9/5 will give the right answer though. I believe the correct solution is to convert the number to a string with the correct number of decimal places, using `toFixed`:

```
> (42 * 1.8)
< 75.60000000000001
> (42 * 1.8).toFixed(2)
< '75.60'
> (42 * 1.8).toFixed(1)
< '75.6'

Reply:
As you said, `There's no guarantee that doing *9/5 will give the right answer though` i don't think it's a good idea

using only `toFixed` can result a different inacurate like this:
```javascript
(1.005).toFixed(2) # 1.00
```

Reply:
Yes you're right:

```
> (1.006).toFixed(2)
< '1.01'
> (1.005).toFixed(2)
< '1.00'
> (1.00501).toFixed(2)
< '1.01'
```

and yet:

```
> (1.05).toFixed(1)
< '1.1'
```

Floating point is [broken](https://0.30000000000000004.com/), and Javascript especially so :-(

However I think toFixed is OK here.  I just want never to see a number with spurious 13 places of decimals.

Reply:
I see what did you mean. Floating point is the evil :-(

also `toFixed` will do this:
```javascript
(1).toFixed(1) // "1.0 "
Math.round((1 + Number.EPSILON) * 10) / 10; //  1
````
this will add `.0` for integers.

Anyway, I think my solution fix the bug we had and maybe different approch can be achived in other PR :)